WalkingChallenge/Leaderboard.swift

```diff
@@ -77,8 +77,7 @@ extension LeaderBoardCell: ConfigurableUITableViewCell {
     standing.text = "\(info.standing)."
     name.text = info.name
 
-    // TODO(compnerd) localise this properly
-    distance.text = "\(info.distance) miles"
+    distance.text = "\(info.distance) " + NSLocalizedString("WCF_KEY_MILES", comment: "miles key")
```

Comment:
I think that we should actually use `MapKit` instead.  It has a `MKDistanceFormatter` that we can use to normalise the distance to a locale specific value.